File: frontend/src/models/contact/ContactRequest.ts
export class ContactRequest {
  subject!: string;
  message!: string;
  emailTo!: string;
  name!: string;


  constructor(subject: string, message: string, emailTo: string, name: string) {
    this.subject = subject;
    this.message = message;
    this.emailTo = emailTo;
    this.name = name;
  }
}

File: backend/mail-service/src/main/java/com/example/mailservice/mailMenager/api/request/ContactRequest.java
package com.example.mailservice.mailMenager.api.request;

import lombok.*;

import javax.validation.constraints.Email;
import javax.validation.constraints.NotBlank;
import javax.validation.constraints.Size;

@Getter
@Setter
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class ContactRequest {
    @NotBlank
    @Size(min = 5, max = 100)
    private String subject;

    @NotBlank
    @Size(min = 5, max = 100)
    private String name;

    @NotBlank
    @Email
    private String emailTo;

    @NotBlank
    @Size(min = 3, max = 1000)
    private String message;
}
